templates/case-study/lucid-aws-fedramp-compliance-case-study.html

```diff
@@ -217,7 +217,7 @@
             width="2464",
             height="1027",
             hi_def=True,
-            loading="auto|lazy") | safe
+            loading="lazy") | safe
```

Comment:
We should add the image classes to the image contents on this page, similarly to what is done on the Ubuntu Pro case study images. There is another image on line 144 that should be updated as well
```suggestion
            loading="lazy"),
            attrs={"class": "p-image-container__image"}) | safe
```